### ISSUE TITLE
Why Can't I Install OpenEDR backend DIRECTLY on my favorite Linux Desktop?

### ISSUE THREAD

**Issue by jymcheong** [jymcheong/OpenEDR#15]
TL;DR - you can but I will have challenges to support...

First & foremost, thank you taking time to explore OpenEDR. I understand your desire to run directly on desktop. 

**Unfortunately, there are really too much permutations on linux desktops that create issues that I will not be able to support.**

In this entry I will cite a few key reasons:

## Installation may change your existing Docker & Other System Dependencies
If you on a developer machine, you may already have docker in place. The installation script may overwrite & mess up other containers &/or development tools within your setup. 

## Desktop tend to reboot more often than server
Restarting the entire backend stack requires stages & when not properly executed, it may corrupt the OrientDB data files. 

In a CyberRange & production scenario, we try to avoid restarting too often even though the whole event-flow is designed to mitigate event losses due to network disruption, or even shutting down of sftp container. 

There's a directory of helper scripts to stop & start backend properly.

## Challenging to Re-Create your environment
Desktops are very personal platform, especially for Linux folks who have the know-how to customize it to their liking. As such, I will not be able to re-create the kind of environment that you have. 

However, that said. If there are certain hardening at your Ubuntu server-side & it ends up not running. Please drop an issue here & I will try my best to assist.

Regards,
Jym
